run-videochat.sh

```diff
@@ -320,7 +322,10 @@ if [ -z $IP ]; then
         error "adb is available, but the phone is not plugged in.\nConnect your phone to USB and allow usb debugging under developer settings or use Wi-Fi (slower)."
     fi
     if ss -ln src :$PORT | grep -q :$PORT; then
-        error "Your port $PORT seems to be in use: try using Wi-Fi.\nIf you would like to use USB forwarding, please free it up and try again."
+	killall adb
```

Comment:
I'd check that it is adb taking up this port and then ask for confirmation for restart adb itself, rather than unconditionally killing the process.